README.md

```diff
@@ -43,41 +35,45 @@ Debug builds are going to be very slow.
 Run
 
 ```shell
-cargo install --git https://github.com/rustformers/llama-rs llama-cli
+cargo install --git https://github.com/rustformers/llama-rs llm-cli
 ```
 
-to install `llama-cli` to your Cargo `bin` directory, which `rustup` is likely to
+to install `llm-cli` to your Cargo `bin` directory, which `rustup` is likely to
 have added to your `PATH`.
 
-It can then be run through `llama-cli`.
+The CLI application can then be run through `llm-cli`.
+
+![Gif showcasing language generation using llama-rs](./doc/resources/llama_gif.gif)
 
 ### Building from repository
 
-Clone the repository, and then build it through
+Clone the repository and then build it with
 
 ```shell
-cargo build --release --bin llama-cli
+git clone --recurse-submodules git@github.com:rustformers/llama-rs.git
+cargo build --release
 ```
 
-The resulting binary will be at `target/release/llama-cli[.exe]`.
+The resulting binary will be at `target/release/llm-cli[.exe]`.
```

Comment:
I think the CLI is called `llm`? The crate is `llm-cli`, but the binary itself is `llm`.

Reply:
Good catch!